admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,328 @@
+          path: /var/log/
+      - name: config # Define ConfigMap volume
+        configMap:
+          name: node-problem-detector-config
+----
++
+<1> Overwrite the *_config/_* directory with the *ConfigMap* volume.
+
+. If you have a node-problem-detector running, delete it as the following:
++
+----
+oc delete -f node-problem-detector.yaml 
+----
+
+. Create the node problem detector with the new yaml file:
++
+----
+oc create -f node-problem-detector.yaml
+----
+
+[NOTE]
+====
+The add-on manager does not support configuration maps. If you are running the node problem detector running as cluster add-on, 
+configuration overwriting is not supported.
+====
+
+[[admin-guide-node-problem-detector-standalone]]
+=== Starting the Node Problem Detector as Standalone
+
+To run node problem detector as a standalone system daemon, execute the following command, pointing to the IP address of your access API server:
+
+----
+# oc node-problem-detector --apiserver-override=http://APISERVER_IP:APISERVER_INSECURE_PORT?inClusterConfig=false
+----
+
+[NOTE]
+====
+Standalone mode uses an insecure API server connection.
+====
+
+
+[[admin-guide-node-problem-detector-gce]]
+== Running the Node Problem Detector in a GCE Cluster
+
+The node problem detector runs as a cluster add-on in a xref:../install_config/configuring_gce.adoc#install-config-configuring-gce[Google Compute Engine (GCE) cluster] 
+and is enabled as a link:https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/node-problem-detector[Kubernetes Add-on] by default.
+
+You can disable the detector by setting the `KUBE_ENABLE_NODE_PROBLEM_DETECTOR` environment variable  in the *_kubernetes/cluster/gce/config-default.sh_* file to false:
+
+[source,yaml]
+----
+KUBE_ENABLE_NODE_PROBLEM_DETECTOR=false
+----
+
+[[admin-guide-node-problem-detector-log]]
+=== Using Custom Logging Tools 
+
+The node problem detector supports file-based kernel logging. 
+
+You can use other file-based logging tools, such as link:https://github.com/kubernetes/node-problem-detector/tree/master/pkg/systemlogmonitor#system-log-monitor[System Log Monitor].
+Use the following command with a comma-separated list of paths to system log monitor configuration files:
+
+----
+# oc node-problem-detector --system-log-monitors <path to config files>
+----
+
+For example:
+
+----
+# oc node-problem-detector --system-log-monitors=config/kernel-monitor.json
+---- 
+
+[NOTE]
+====
+The node problem detector does not detect problems reported by log tools, such as journald or rsyslogd.
+====
+
+The node problem detector starts a separate log monitor for each configuration. You can use different log monitors to monitor different system logs.
+
+[[admin-guide-node-problem-detector-monitor]]
+=== Using Custom Monitoring Tools 
+
+You can use other montoring plugin tools, such as 
+link:https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/node_problem_detector_linux.go[node_problem_detector_linux.go
+].
+
```

Comment:
@joelsmith I found this link to a purported custom monitor plug-in from an [Kubernetes issue on adding custom plug-ins](https://github.com/kubernetes/node-problem-detector/issues/150#issuecomment-346488233). Is this a good example?